packages/react/src/transactions/index.ts

```diff
@@ -21,3 +22,9 @@ export type {
   NoCollectPolicy,
   SupportedPublicationMediaType,
 } from '@lens-protocol/domain/use-cases/publications';
+export type {
+  ChargeFollowPolicy,
+  NoFeeFollowPolicy,
+} from '@lens-protocol/domain/use-cases/profile';
+export { FollowPolicyType } from '@lens-protocol/domain/use-cases/profile';
+export { getFollowPolicyTypeFromProfileFieldsFragment } from '@lens-protocol/api-bindings';
```

Comment:
It's a bit of a mouthful. Should we provide a local-only field or something instead?